src/PKSim.CLI.Core/Services/QualificationRunner.cs

```diff
@@ -91,7 +91,7 @@ public async Task RunBatchAsync(QualificationRunOptions runOptions)
          }
 
          var begin = DateTime.UtcNow;
-         var project = await _snapshotTask.LoadProjectFromSnapshot(snapshot);
+         var project = await _snapshotTask.LoadProjectFromSnapshot(snapshot, runOptions.Run);
```

Comment:
@Yuri05 I have added the option to decide whether simulation should be run or not in the option. By default, it will now be NOT run